js/bars.js

```diff
@@ -778,6 +796,20 @@ historian.Bars.prototype.renderSeries_ = function(data) {
             }
             return series.color(eventType);
         }
+        // Color for sensor historian is specifically chosen.
+        if (series.source == 
```

Comment:
Shouldn't you be using group.source here?

Reply:
Series.source is correct because line 702 - 798 are all code for the switch cases for the color function.